addons/web/static/src/views/model.js

```diff
@@ -112,6 +112,7 @@ export function useModel(ModelClass, params, options = {}) {
             ? globalState.useSampleModel
             : component.props.useSampleModel
     );
+    model.isSampleModelActive = false;
```

Comment:
since new gantt merge, we have a key isSample set on orm in buildSampleORM

Reply:
Okay thanks. 🙂